modules/rosa-sts-about-ocm-role.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// rosa_planning/rosa-sts-ocm-role.adoc
+//
+:_content-type: CONCEPT
+[id="rosa-sts-about-ocm-role_{context}"]
+= About the ocm-role IAM resource
+
+You need to create the `ocm-role` IAM resource to enable a Red Hat organization of users to create ROSA clusters.
+
+Some considerations for your `ocm-role` IAM resource are:
+
+* You have only one `ocm-role` per Red Hat organization.
```

Comment:
You can have more than one `ocm-role` per org, as long as they are for different AWS accounts.